tracing-attributes/src/expand.rs

```diff
@@ -191,8 +191,18 @@ fn gen_block<B: ToTokens>(
     })();
 
     let err_event = match args.err_mode {
-        Some(ErrorMode::Display) => Some(quote!(tracing::error!(error = %e))),
-        Some(ErrorMode::Debug) => Some(quote!(tracing::error!(error = ?e))),
+        Some(FormatMode::Default | FormatMode::Display) => {
```

Comment:
This syntax isn't accepted on Rust 1.42.0, our current minimum supported Rust version (MSRV): https://github.com/tokio-rs/tracing/runs/4280091568?check_suite_focus=true

In order to fix the build on 1.42.0, we'll need to write this instead:
```suggestion
        Some(FormatMode::Default) | Some(FormatMode::Display) => {
```